### PR TITLE
KOGITO-7210: Fix swf editor missing module error

### PR DESCRIPTION
**Pull request by tomasdavidorg**
https://issues.redhat.com/browse/KOGITO-7210

### CODE REVIEW

File: packages/serverless-workflow-editor/package.json [hunk 1]
```diff
@@ -36,9 +36,12 @@
     "@kie-tools-core/operating-system": "0.0.0",
     "@kie-tools-core/editor": "0.0.0",
     "@kie-tools-core/patternfly-base": "0.0.0",
+    "@kie-tools-core/webpack-base": "0.0.0",
```
Comment (caponetto) [inline packages/serverless-workflow-editor/package.json]: I think this can go to `devDependencies`. :)

Comment (tomasdavidorg) [reply packages/serverless-workflow-editor/package.json]: Thanks @caponetto ... it should be ok now. 🙂

Comment (caponetto) [reply packages/serverless-workflow-editor/package.json]: Almost there hehe `packages/build-env/graph.dot` needs update as well.

Comment (caponetto) [reply packages/serverless-workflow-editor/package.json]: @tomasdavidorg  ^

**Review by caponetto — APPROVED** [review]

**Review by tiagobento — APPROVED** [review]

**Pull request merged**